doc/storage_ui.md

```diff
@@ -33,22 +54,20 @@ and resizing existing partitions to make space for the new system. It decides by
 partitions should be affected following a logic that, even though is configurable in the control
 file and by the UI, is hard to follow for many end users.
 
-The proposal described in this document would not rely on that logic. Instead, Agama would
-only propose two ways of making space:
+### Reusing LVM Setups
 
-- Completely wipe the content of the selected disk(s) and start from scratch.
-- Manually select which partitions to keep / delete / resize.
+For historical reasons, YaST tries to reuse existing LVM volume groups when making a proposal. That
+behavior can be very confusing in many situations. To avoid the associated problems, the Agama
+storage proposal will not automatically reuse existing LVM structures.
 
 ### About the Initial Proposal
 
 Currently YaST tries really hard to present an initial proposal to the user, even if that implies
 several subsequent executions of the `GuidedProposal`, each of them with a less ambitious
-configuration. For that it relies on two features of the so-called volumes (i.e. the specification
-of the partitions or LVM logical volumes to be created during the process).
+configuration. For that it relies on two features of the so-called volumes (volumes are already
+described in a previous section).
 
-- First of all, every volume specifies two different kinds of lower size limits in the control file.
-  The so-called "desired size" that is the smallest size that is recommeded for a normal usage of
-  that volume and the "min size" that is the lower threshold for the volume to be minimally useful.
+- First of all, every volume specify both a "min size" and a "desired size" (as already explained).
```

Comment:
NP: I would mention the section ("see Volumes in the YaST Proposal: Lessons Learned") instead of saying "already explained".